WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -588,6 +591,8 @@ class MediaCoordinator: NSObject {
         case ended
         case failed(error: NSError)
         case progress(value: Double)
+        /// The upload was cancelled by the user.
+        case cancelled
```

Comment:
[nit] Let's use the US spelling for canceled
```suggestion
        case canceled
```

Reply:
I think it's a toss-up. Apple uses the double "LL" version throughout their APIs too. Example: https://developer.apple.com/documentation/swift/task/iscancelled-swift.type.property